src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -28,7 +28,7 @@
         "symfony/property-info": "~2.8|3.0",
         "symfony/security": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
-        "symfony/validator": "~2.8|~3.0",
+        "symfony/validator": "~3.1",
```

Comment:
This is an issue: we won't test the bridge with symfony/validator anymore, which could hide BC breaks.
We need to find a way to keep ~2.8 here.
(same comment for the other cases below)